controller/schema.mustache

```diff
@@ -12,5 +13,22 @@
       "faker": "name.findName"
     }
   },
+  <%#relationships%>
+  "<%pluralName%>": {
```

Comment:
Sorry, this was a total driveby comment by me. The relevant part of the spec is here:

http://json-schema.org/latest/json-schema-hypermedia.html#anchor3

That should give you some context on why it's the singular rather than the plural rather than just "That aint right".